xbmc/Application.cpp

```diff
@@ -4085,16 +4085,24 @@ PlayBackRet CApplication::PlayFile(const CFileItem& item, bool bRestart)
       CJobManager::GetInstance().PauseJobs();
     }
 
-    // don't hold graphicscontext here since player
-    // may wait on another thread, that requires gfx
-    CSingleExit ex(g_graphicsContext);
+    if (item.Exists()) // this is also an implicit call to CWakeOnAccess::WakeUpHost() if enabled
+    {
+      // don't hold graphicscontext here since player
+      // may wait on another thread, that requires gfx
+      CSingleExit ex(g_graphicsContext);
 
-    iResult = m_pPlayer->OpenFile(item, options);
+      iResult = m_pPlayer->OpenFile(item, options);
+    }
+    else
+    {
+      CGUIDialogKaiToast::QueueNotification(CGUIDialogKaiToast::Error, 
```

Comment:
Ignore this, this is not blocking anything. As it's a very very specific case where it could happen. Don't worry.